src/System.Diagnostics.Process/tests/ProcessTests.netcoreapp.cs

```diff
@@ -119,5 +125,182 @@ public void BothArgumentSetAndArgumentListSet()
             testProcess.StartInfo = psi;
             Assert.Throws<InvalidOperationException>(() => testProcess.Start());
         }
+
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.Uap)]
+        public void VerifyingCiBehavior()
+        {
+            throw new Exception();
```

Comment:
@danmosemsft, any idea why this doesn't cause the Uap test run to report a failure? (I modeled this method's attributes after https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs#L269-L271)

I'd like to verify that failures are being reported by the CI system before I implement this PR's one UAP-specific test. :-)

Thanks!

Reply:
@ViktorHofer can you help?

Reply:
I need to look into that. Will respond later today.

Reply:
Can't repro locally. Opened to check CI integrity: https://github.com/dotnet/corefx/pull/32022

Reply:
Failing on CI too: https://mc.dot.net/#/user/ViktorHofer/pr~2Fjenkins~2Fdotnet~2Fcorefx~2Fmaster~2F/test~2Ffunctional~2Fcli~2F/99d178a2fa1bbba0b540ba83f94dbd70902759f0/workItem/System.Text.RegularExpressions.Tests

Reply:
@ViktorHofer, does this mean the test should now fail for me?

Reply:
I verified in a separate PR that the CI reports the failing test correctly. I didn't commit anything into master related to that. There must be some other reason why the test here isn't reported a failure, i.e. method not being invoked, conditional compilation. I haven't yet checked your changes.

Reply:
@ViktorHofer, do you have time to take a peak? I'm stumped by this.

Reply:
@danmosemsft, do you have anyone who could help with this?

Reply:
@bgribaudo 	it would have to be @ViktorHofer 	but it might be a few days.

Reply:
yes I will take a look today.

Reply:
Most tests aren't running for uap because of this: https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.Process/tests/ProcessTests.cs#L19. I will re-enables ones that work fine and will either commit into this PR or create another one.